README.md

```diff
@@ -37,3 +37,7 @@ Team Shooting Star focuses on polishing the tutorial and project content, as wel
 ### Team Telescope
 
 Team Telescope focuses on measuring the overall impact of the tool by implementing well defined metrics and dashboards. Team members will work with both developers and users to assess the tool, utilizing data and user feedback to suggest improvements.
+
+## Media
```

Comment:
Instead of `Media` do we want to say something like Community?

Reply:
I'm okay with that change! Should I also add the Operate First Slack or other community outlets? Or just the YT since everything else is Operate First?

Reply:
I think just YT should be enough for now, since our chat room is in GChat and not really on slack